tests/python_functional/src/helpers/snmptrapd/conftest.py

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env python
+#############################################################################
+# Copyright (c) 2020 One Identity
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of the GNU General Public License version 2 as published
+# by the Free Software Foundation, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301  USA
+#
+# As an additional exemption you are allowed to compile & link against the
+# OpenSSL libraries as published by the OpenSSL project. See the file
+# COPYING for details.
+#
+#############################################################################
+import os
+
+import pytest
+from pathlib2 import Path
+from psutil import TimeoutExpired
+
+import src.testcase_parameters.testcase_parameters as tc_parameters
+from src.driver_io.file.file_io import FileIO
+from src.executors.process_executor import ProcessExecutor
+from src.syslog_ng_config.statements.destinations.destination_reader import DestinationReader
+
+
+class SNMPtrapd(object):
+    def __init__(self, port):
+        self.snmptrapd_proc = None
```

Comment:
We might want to add a `which snmptrapd` check, to make sure we have the necessary binary, or send a human readable error message.

Reply:
that's where we were originally. now we concluded that if snmptrapd is not available we will throw an exception. The text of exception is pretty clear and I can display almost the same message.

Reply:
Yep, that's good enough :)